crates/api_crud/src/private_message/create.rs

```diff
@@ -91,11 +91,11 @@ impl PerformCrud for CreatePrivateMessage {
       let inbox_link = format!("{}/inbox", context.settings().get_protocol_and_hostname());
       send_email_to_user(
         &local_recipient,
-        &lang.notification_private_message_subject(&local_recipient.person.name),
+        &lang.notification_private_message_subject(&local_user_view.person.name),
```

Comment:
How about renaming `local_user_view` to `sender` to avoid confusion? Although that would make it inconsistent with other api handlers which all use `local_user_view`.

Reply:
i'm good with that, or i could could do something like `let sender_name = &local_user_view.person.name` and use that in both places?

Reply:
That sounds good to me.

Reply:
done!